.github/workflows/lock.yaml

```diff
@@ -0,0 +1,23 @@
+name: Lock inactive closed issues
+# Lock closed issues that have not received any further activity for two weeks.
+# This does not close open issues, only humans may do that. It is easier to
+# respond to new issues with fresh examples rather than continuing discussions
+# on old issues.
```

Comment:
I'm personally 👎 down on this policy, especially for a low-volume repo like this one... I'd rather let folks be able to swing by later and say "for anyone googling, here's new/updated info"... I understand why some busy projects choose to do it, but the problem it solves doesn't exist in this project. 

Just my $0.02, not something I feel strongly about.

Reply:
That's reasonable. Long term, as we've discussed elsewhere, hopefully we can also distribute the work of maintaining across all these projects to dedicated sub-teams as it's unreasonable to expect you to keep up with everything... you're already one of the more productive OSS maintainers I know just with what you accomplish with the core Pallets stuff.